stdlib/REPL/test/replcompletions.jl

```diff
@@ -1498,7 +1498,7 @@ end
     @test "length=" ∈ c
     @test "len2=" ∈ c
     c, r = test_complete_foo("kwtest3(1+3im; named")
-    @test "named" ∈ c
+    @test "named" ∉ c
     # TODO: @test "namedarg=" ∉ c
```

Comment:
@Liozou is this TODO relevant? I wasn't sure what it meant

Reply:
Ah so this TODO is meant to be fixed by https://github.com/JuliaLang/julia/pull/44434, like the other TODOs introduced in https://github.com/JuliaLang/julia/pull/43536. It means that `kwtest3(1+3im; named#TAB` currently suggests `namedarg=` as a suggestion, but that's actually invalid since the method of `kwtest3` that accepts the `namedarg` keyword argument cannot take a `Complex` as its first argument.
I made these cases TODOs instead of `@test_broken` since they are wishful features really, I call them "invalid" but they are not regressions. So I believe the TODO is still relevant, but it's orthogonal to this PR.